package.json

```diff
@@ -33,7 +33,6 @@
     "node": ">=10.18"
   },
   "scripts": {
-    "postinstall": "patch-package",
```

Comment:
I removed this because it was focused on nullable which is gone now. We might need to bring it back to support OAS3.0 but im on a rampage trying to get 3.1 working first.